warehouse/integrations/bigquery/bigquery.go

```diff
@@ -443,44 +441,19 @@ func (bq *BigQuery) loadTable(ctx context.Context, tableName string) (
 	)
 	log.Infon("started loading")
 
-	loadFileLocations, err := bq.loadFileLocations(ctx, tableName)
+	objectLocation, err := bq.uploader.GetSampleLoadFileLocation(ctx, tableName)
```

Comment:
noob question - does a folder only contain files or can it have subfolders as well ?

Reply:
@achettyiitr Also would dir path contain files only related to this load job? Can you share example of full paths?

Reply:
https://www.notion.so/rudderstacks/Slaves-5fed5a5522984924879fef97055f582b?pvs=4#9c3079b24afd4707b1859c2c6b797306

Everytime time load generation happens it will generate a new `<uuid>-<table>` folder, within which all files will present. This folder will contain only the files, it will not contain any folder within it.
`<prefix>/rudder-warehouse-load-objects/<table>/<source_id>/<uuid>-<table>/<timestamp>.<source_id>.<uuid>.<table>.<uuid>.<extension>`